### DISCUSSION TITLE
With Fleet API, what to enter as a Refresh Token?

### DISCUSSION THREAD

**Discussion by eigenphase** [alandtse/tesla#1020] [Q&A]
I have the same issue as in https://github.com/alandtse/tesla/discussions/950 but this topic has been closed and I disagree with the "solution".

In this extensions' configuration, what am I supposed to enter as "Refresh Token" when using the Fleet API? 

<img width="323" alt="image" src="https://github.com/user-attachments/assets/130d211b-2d25-4985-8c11-e830db9d1ed9">

**Comment by eigenphase** [comment]
For everyone with the same question: Contrary to what it is saying it does **not need the Auth for Tesla app any more**! It is the Refresh Token that's spit out using the "Generate token from URL" button on the "Open Web UI" from the tesla proxy.

**Reply by madsdyd** [reply]
Thank you, segdy. I was struggling with this for 45 minutes or so.
